scripts/motors_test.py

```diff
@@ -24,7 +22,15 @@ def constrain(x, l, h):
     command_pub = rospy.Publisher('uav_direction_command', OrientationThrottleStamped, queue_size=0)
     arm_pub = rospy.Publisher('uav_arm', BoolStamped, queue_size=0)
```

Comment:
We should get rid of this if we aren't using it, and also call `rospy.wait_for_service('uav_arm')` before we create a `ServiceProxy`.